front/src/store/index.js

```diff
@@ -25,7 +26,16 @@ export default new Vuex.Store({
     },
     CLEAR_USER_DATA (state) {
       localStorage.removeItem('user');
+      delete axios.defaults.headers.common.Authorization;
```

Comment:
this was already in the dev branch before me. I only added deleting the header because it very much spoils my work(if you try to restore the password after logging out of the account, the server will not understand where the user is going and the request is ignored)